bin/fixbase/gwfixbase.ml

```diff
@@ -178,6 +183,7 @@ let fevents_witnesses = ref false
 let marriage_divorce = ref false
 let invalid_utf8 = ref false
 let key = ref false
+let utf8_key = ref false
```

Comment:
I don't understand the name, why is it "_key"?

Reply:
Fixbase.fix_key does not detect conflicts arising with ' and ‘ and ’ in names.
Fixbase.scan_utf8_conflicts does.
I did not try to merge those two functions, but it may be worth a try

Reply:
I think it is probably best to keep them separated